etm/operator-interface-spec/etm-oper-api.yaml

```diff
@@ -651,15 +829,55 @@ components:
           maxLength: 1000
           type: string
           description:
-            Informative text about the aircraft. Not used by the UTM System.
+            Informative text about the aircraft. Not used by the ETM System.
             Only for human stakeholders.
           default: NOT PROVIDED
         sensitivity_type:
           $ref: "#/components/schemas/SensitivityType"
 
+    ConflictingOperationResponse:
```

Comment:
I understand this is an Operation, not a Response portion of a protocol.   rename to ConflictingOperation